packages/sdk-cli/src/models/DeveloperRelay.ts

```diff
@@ -41,7 +40,14 @@ export default class DeveloperRelay {
   static async create(local = false) {
     if (local) {
       // Create an output directory (if doesn't exist)
-      await fsPromises.mkdir(RELAY_DIRECTORY_PATH);
+      try {
```

Comment:
Creating the directory in the calling code for making a relay instance seems a little odd from an encapsulation perspective. I'd really like to see this PR land soon though, so if this is likely to be non-trivial for you to adjust, please just create a ticket to come back and refactor it.

Reply:
`https://github.com/Fitbit/developer-bridge/pull/176/commits/134923f23e7f0c7945d8ec7e430076a4c5208556`
`https://github.com/Fitbit/developer-bridge/pull/176/commits/1b3add13161c43a7f2fd4d1b125df4c5563972fe`